src/hbase/client/async-batch-rpc-retrying-caller.cc

```diff
@@ -70,7 +70,8 @@ AsyncBatchRpcRetryingCaller<REQ, RESP>::~AsyncBatchRpcRetryingCaller() {}
 template <typename REQ, typename RESP>
 Future<std::vector<Try<RESP>>> AsyncBatchRpcRetryingCaller<REQ, RESP>::Call() {
   GroupAndSend(actions_, 1);
-  return collectAll(action2futures_);
+  // use the executor to convert he SemiFuture to a Future.
+  return std::move(collectAll(action2futures_)).via(cpu_pool_.get());
```

Comment:
whats the need for move() everywhere? Isn't a copy elided with RVO?

Reply:
yeah I'm surprised to see that there too. This is something I'm usually good at picking up in reviews of others' code, so it's kind of embarrassing that I didn't notice this myself, but I got pretty careless when I pulled this commit out from the original PR. Removed!